checkstyle-tester/README.md

```diff
@@ -129,6 +129,12 @@ Example:
 </module>
 ```
 
+##### Modified Grammar Check Regression Report
+
+After modifying Checkstyle's ANTLR grammar, it is neccessary to compare how Checkstyle's checks are affected by your changes.  This is done by splitting the modules and settings in `checkstyle_checks.xml` configuration file, found in the checkstyle repository, into five separate configuration files (this helps avoid OOM).  You may modify all the modules that depend on external files to use default settings.  For each of the five configuration files, you should use the `my_checks.xml` file as a base, and add the modules from `checkstyle_checks.xml` to it. Then `diff.groovy` should be run on all projects in `projects-to-test-on.properties`, using the `diff.groovy` script once for each configuration file.  See instructions below for "Basic Difference Report".  If making changes to Checkstyle's grammar, also see "Testing Abstract Syntax Tree Regression" below.
```

Comment:
> into five separate configuration files 

The more checks we add the bigger the splits will be. I would try to guide on most optimally split method.

> See instructions below for "Basic Difference Report".
> see "Testing Abstract Syntax Tree Regression" below.

Just link to the instructions.

Reply:
Fixed, unless you think I should go into more detail.